core/src/test/java/com/linecorp/armeria/common/HttpStatusTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.server.AbstractHttpService;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.testing.junit.server.ServerExtension;
+
+class HttpStatusTest {
+
+    @RegisterExtension
+    public static final ServerExtension server = new ServerExtension() {
```

Comment:
I don't think you really need a real server to test the behavior of the methods you added. How about removing this `ServerExtension` and just add some simple tests like the following?

```java
@Test
void isSuccess() {
    assertThat(HttpStatus.valueOf(199).isSuccess()).isFalse();
    assertThat(HttpStatus.valueOf(200).isSuccess()).isTrue();
    assertThat(HttpStatus.valueOf(299).isSuccess()).isTrue();
    assertThat(HttpStatus.valueOf(300).isSuccess()).isFalse();
}

...
```

Reply:
Hi Trustin. Simple and better. I will update my PR. Thanks for suggestion